pkg/k8shandler/elasticsearch.go

```diff
@@ -325,14 +328,16 @@ func GetClusterNodeCount(clusterName, namespace string) (int32, error) {
 
 	payload := &esCurlStruct{
 		Method: http.MethodGet,
-		Uri:    "_cluster/health",
+		URI:    "_cluster/health",
 	}
 
 	curlESService(clusterName, namespace, payload)
 
 	nodeCount := int32(0)
-	if payload.ResponseBody["number_of_nodes"] != nil {
-		nodeCount = int32(payload.ResponseBody["number_of_nodes"].(float64))
+	if nodeCountFloat, ok := payload.ResponseBody["number_of_nodes"]; ok {
+		if safeCasted, ok := nodeCountFloat.(int32); ok {
```

Comment:
per the docs for `json.Unmarshal` all numbers are unmarshalled as `float64`. hence the conversion before.